preprocessing_qc/process_cfdna.ipynb

```diff
@@ -0,0 +1,876 @@
+{
```

Comment:
<p>this funct could go in other file with other functions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_atac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes, this is a good point. I am going to change the folder structure of this project once I get more cfDNA data. When I do that I will make a preprocessing methods file to store all the methods.</p>